src/main/drivers/bus_spi_hal.c

```diff
@@ -295,10 +295,10 @@ static bool spiBusReadBuffer(const busDevice_t *bus, uint8_t *out, int len)
 }
 
 // return uint8_t value or -1 when failure
```

Comment:
It would make more sense if return value is `int` (like `getc()`). But as currently implemented, SPI should not detect error (IMO timeouts in HAL are meant for slave mode)